README.md

```diff
@@ -35,7 +36,7 @@ If you find that it is not, please [file an issue](https://github.com/cfpb/wagta
 pip install wagtail-treemodeladmin
 ```
 
-2. Add `treemodeladmin` (and `wagtail.contrib.modeladmin` if it's not already) as an installed app in your Django `settings.py`:
+2. Add `treemodeladmin` (and `wagtail-modeladmin` if it's not already) as an installed app in your Django `settings.py`:
```

Comment:
```suggestion
2. Add `treemodeladmin` (and `wagtail_modeladmin` if it's not already) as an installed app in your Django `settings.py`:
```

Also below line still shows `'wagtail.contrib.modeladmin'` in the list of `INSTALLED_APPS`, should be `'wagtail_modeladmin'` (underscore, not hyphen) as well:

https://github.com/cfpb/wagtail-treemodeladmin/blob/78d31c35604713ac77c64fe7b6769cc989292ace/README.md?plain=1#L44